### DISCUSSION TITLE
How to accelerate the loading speed of constraints?

### DISCUSSION THREAD

**Discussion by Sswd** [yalmip/YALMIP#1440] [General discussion]
"As shown in the figure, I am using a multi-layer nested for-loop to load constraints. There are more than a dozen programs like the one shown, and it takes more than an hour to load before the solver starts solving. I tried using the parfor loop method to solve this issue, but after testing and browsing through your relevant answers, I realize that this is not feasible. I would like to ask if there are any methods to resolve this issue?""The number of constraints is approximately fifty to sixty thousand."
![6a8d3f71-67f8-4c1a-a5e2-6b6f4669c342](https://github.com/user-attachments/assets/45c3e936-4b1a-4172-905a-0a4fa7a3e6a4)
![6a8d3f71-67f8-4c1a-a5e2-6b6f4669c342](https://github.com/user-attachments/assets/8dc03883-7398-4098-87ad-c77546a44781)

**Comment by johanlofberg** [comment]
You will have to vectorize it massively
